lib/auth/windows/ldap.go

```diff
@@ -103,6 +103,11 @@ const (
 	AttrOSVersion = "operatingSystemVersion"
 	// AttrPrimaryGroupID is the primary group id of an LDAP object
 	AttrPrimaryGroupID = "primaryGroupID"
+
+	// SearchPageSize is desired page size for LDAP search. In Active Directory the default search size limit is 1000 entries,
+	// so in most cases the 1000 search page size will result in the optiomal amount of requests made to
+	// LDAP server.
+	SearchPageSize = 1000
```

Comment:
```suggestion
	searchPageSize = 1000
```

Since this is only used in this package, no need to export it.

We can always export it later if the need arises, but best to start unexported until there's a need to change it.